plugins/main/public/components/management/cluster/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState, useEffect } from 'react';
+import { getCore, getPlugins } from '../../../../kibana-services';
+import { SearchResponse } from '../../../../../../../src/core/server';
+import { IndexPattern } from '../../../../../../../src/plugins/data/common';
+import { I18nProvider } from '@osd/i18n/react';
+import useSearchBar from '../../../common/search-bar/use-search-bar';
+import {
+  ErrorFactory,
+  ErrorHandler,
+  HttpError,
+} from '../../../../react-services/error-management';
+import { LoadingSpinner } from '../../../common/loading-spinner/loading-spinner';
+import { withErrorBoundary } from '../../../common/hocs/error-boundary/with-error-boundary';
+import { EuiSpacer, EuiFlexItem } from '@elastic/eui';
+
+import { OverviewCards } from '../components/overview_cards';
+import { endpointSummary } from '../../../../utils/applications';
+import { WzRequest } from '../../../../react-services';
+import { ConfigurationCards } from '../components/configuration_cards';
+import { NodeList } from '../node-list';
+import {
+  AlertsDataSource,
+  AlertsDataSourceRepository,
+  PatternDataSource,
+  tParsedIndexPattern,
+  useDataSource,
+} from '../../../common/data-source';
+
+const SearchBar = getPlugins().data.ui.SearchBar;
+
+interface DashboardCTProps {
+  statusRunning: string;
+}
+
+interface ClusterDashboardState {
+  showConfig: boolean;
+  showNodes: boolean;
+  nodeList: any;
+  configuration: any;
+  version: any;
+  nodesCount: number;
+  agentsCount: number;
+}
+
+const DashboardCT: React.FC<DashboardCTProps> = ({ statusRunning }) => {
+  const {
+    filters,
+    dataSource,
+    fetchFilters,
+    isLoading: isDataSourceLoading,
+    fetchData,
+    setFilters,
+  } = useDataSource<tParsedIndexPattern, PatternDataSource>({
+    DataSource: AlertsDataSource,
+    repository: new AlertsDataSourceRepository(),
+  });
+
+  const [state, setState] = useState<ClusterDashboardState>({
+    showConfig: false,
+    showNodes: false,
+    nodeList: [],
+    configuration: undefined,
+    version: undefined,
+    nodesCount: 0,
+    agentsCount: 0,
+  });
+
+  const [results, setResults] = useState<SearchResponse>({} as SearchResponse);
+
+  const { searchBarProps } = useSearchBar({
+    indexPattern: dataSource?.indexPattern as IndexPattern,
+    filters,
+    setFilters,
+  });
+  const { query, dateRangeFrom, dateRangeTo } = searchBarProps;
+
+  useEffect(() => {
+    if (isDataSourceLoading) {
+      return;
+    }
+    fetchData({
+      query,
+      dateRange: {
+        from: dateRangeFrom,
+        to: dateRangeTo,
+      },
+    })
+      .then(results => {
+        setResults(results);
+      })
+      .catch(error => {
+        const searchError = ErrorFactory.create(HttpError, {
+          error,
+          message: 'Error fetching vulnerabilities',
+        });
+        ErrorHandler.handleError(searchError);
+      });
+  }, [JSON.stringify(fetchFilters), JSON.stringify(query)]);
```

Comment:
**nitpick**: the values of date range seems not to be considered.

Reply:
`dateRangeFrom` and `dateRangeTo` are added to the useE dependency array